lib/gds_api/exceptions.rb

```diff
@@ -52,8 +53,8 @@ def ignoring_missing(&block)
       ignoring([HTTPNotFound, HTTPGone], &block)
     end
 
-    def build_specific_http_error(error, url, details = nil)
-      message = "url: #{url}\n#{error.http_body}"
+    def build_specific_http_error(error, url, details = nil, request_body = nil)
+      message = "URL: #{url} Response body: #{error.http_body} Request body: #{request_body}"
```

Comment:
I can reintroduce the line break, though Errbit stripped that out when it displays the error.